docs/CI_CD_WORKFLOWS.md

```
@@ -0,0 +1,893 @@
+        uses: zaproxy/action-baseline@v0.7.0
+        with:
+          target: 'http://localhost:3000'
+          rules_file_name: '.zap/rules.tsv'
+          cmd_options: '-a'
+```
+
+## Workflow Optimization
+
+### Caching Strategy
+
+#### Dependency Caching
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)